plugins/decklink/DecklinkInput.hpp

```diff
@@ -60,8 +39,8 @@ class DeckLink {
 		channelFormat = format;
 	}
 
-	bool Activate(DeckLinkDevice *device, long long modeId);
-	void Deactivate();
-
 	bool buffering = false;
+
+    bool Activate(DeckLinkDevice *device, long long modeId) override ;
```

Comment:
Style:  Use tabs to indent.

Did these declarations need to be moved, or could they be edited in place?